Developing-new-features-with-feature-gating.md

```diff
@@ -49,6 +86,8 @@ Features fall in the three types of stages: dev, test and prod. In short, the st
 - test feature can be enabled in dev or test environments, but it can never be enabled in production environments.
 - prod feature can be enabled in any of the dev, test, production environments.
 
+Note: The environment the flag is placed in determines the enviroment(s) where it CAN BE enabled, not where it IS enabled. A feature in `test` stage, as mentioned above, can only be enabled in the `dev` and `test` environments, and not in the `prod` environment. By default, however, it is disabled in all environments. See the [Changing Value of Feature Flags](#changing-value-of-feature-flags) section for details on how to enable/disable feature flags.
```

Comment:
When you say "it is disabled in all environments", what is "it" and what does the "default" state refer to exactly?

Reply:
it: The flag
default state: when the flag is first added to the codebase.

Modified the doc to include these details.